hedera/management/commands/reset_db.py

```diff
@@ -0,0 +1,60 @@
+from django.core.management.base import BaseCommand
+
+from lattices.models import FormNode, LatticeNode, LemmaNode
+from lemmatized_text.models import (
+    LemmatizationLog,
+    LemmatizedText,
+    LemmatizedTextBookmark
+)
+from vocab_list.models import (
+    PersonalVocabularyList,
+    PersonalVocabularyListEntry,
+    PersonalVocabularyStats,
+    VocabularyList,
+    VocabularyListEntry
+)
+
+
+MODELS = [
+    LemmatizedText,
+    LatticeNode,
+    VocabularyList,
+    PersonalVocabularyList,
+    FormNode,
+    LemmaNode,
+    LemmatizationLog,
+    LemmatizedTextBookmark,
+    VocabularyListEntry,
+    PersonalVocabularyListEntry,
+    PersonalVocabularyStats
+]
+
+
+class Command(BaseCommand):
```

Comment:
Awesome! Will refer to this in the future for any situation calling for management commands!